mod-di-converter-storage-server/src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -96,7 +96,6 @@ Future<Integer> loadData(TenantAttributes attributes, String tenantId, Map<Strin
         .compose(m -> runSqlScript(DEFAULT_QM_AUTHORITY_UPDATE_JOB_PROFILE, headers, context))
         .compose(m -> runSqlScript(DEFAULT_QM_MARC_BIB_UPDATE_JOB_PROFILE, headers, context))
         .compose(m -> runSqlScript(DEFAULT_QM_HOLDINGS_UPDATE_JOB_PROFILE, headers, context))
-        .compose(m -> profileMigrationService.migrateDataImportProfiles(headers, context))
```

Comment:
if this is removed, where else is it being executed?

Reply:
as far as I understood, it can be safely removed, because this migration is applicable for initializing wrappers, and not going to be executed again, as there a table called metadata_internal, which checks whether migration has taken place, 
migration related association (associations-migration-v2/associations_migration.sql) is going to be called from schema.json file

Reply:
in the next pr, we are going to remove the migration service completely

Reply:
Hi @okolawole-ebsco, our logic here is the following
1. Wrappers are introduced in Poppy, "wrappers migration" is needed for Orchid -> Poppy profiles and for clean Poppy/Quesnelia deployment for default profiles, scripts for which were written prior to wrappers.
2. Quesnelia -> Ramson upgrade will require "associations migration" for associations that already use wrappers. Clean Ramsons deployment will trigger creation of default profiles, and we updated those scripts to use wrappers and the new associations table structure.
3. As a result, it looks like we don't need the "wrappers migration" service anymore as it will create duplicate wrappers for default profiles. Therefore, we removed the call to this migration service in this PR. But we are not cleaning old associations tables and the migration service until we tested all possible migrations.
Do you see any problems in such approach?

Reply:
@JavokhirAbdullayev Thanks!

@KaterynaSenchenko if I am Orchid version and I want to upgrade to Ramsons, will my data be consistent?

Reply:
in association_migration file, while  iterating old associations, we are creating new associations, if conflict do nothing

Reply:
It appears that I am wrong about the 3d point in my comment. This call for migration service should not take any effect on data set that already migrated on wrappers, we can leave this call as is for now. In that case migration from Orchid to Ramsons should also be possible

Reply:
@KaterynaSenchenko Sounds good! @JavokhirAbdullayev Please make appropriate changes.